### ISSUE TITLE
Error when importing an FOB file 

### ISSUE THREAD

**Issue by PatrickGrabensteiner** [HodorNV/ALOps#100]
when i Import an FOB file i get the following Error: "!"
Yesterday we got an Update of ALOps, it has worked before
![image](https://user-images.githubusercontent.com/35697862/77548030-476e5780-6eae-11ea-9147-55a29fe6082e.png)

```
2020-03-25T13:54:44.1489350Z ##[section]Starting: ALOpsFobImport
2020-03-25T13:54:44.1886085Z ==============================================================================
2020-03-25T13:54:44.1886480Z Task         : ALOps Import FOB
2020-03-25T13:54:44.1886751Z Description  : ALOps Import FOB
2020-03-25T13:54:44.1887009Z Version      : 1.428.1141
2020-03-25T13:54:44.1887260Z Author       : Hodor
2020-03-25T13:54:44.1887544Z Help         : Import objects from .FOB file.
2020-03-25T13:54:44.1887922Z ==============================================================================
2020-03-25T13:54:45.7780364Z *** Validate configuration
2020-03-25T13:54:46.6049352Z *** Importing required PS-Functions
2020-03-25T13:54:47.3601228Z *** Connect Docker Session
2020-03-25T13:54:51.6227449Z *** Initiate Docker Session
2020-03-25T13:54:52.3490499Z *** Setup Docker Session
2020-03-25T13:54:52.3728188Z *** Check Generate Symbol Reference
2020-03-25T13:54:52.6668798Z *** Setup database connection
2020-03-25T13:54:53.0779183Z *** Using Windows-Authentication for Database
2020-03-25T13:54:53.1098333Z *** Setup FinSql link
2020-03-25T13:54:58.0285506Z *** Searching for Client-FinSql in Registry
2020-03-25T13:54:58.3677543Z *** Client-FinSql not found in registry: [HKLM:\SOFTWARE\Wow6432Node\Microsoft\Microsoft Dynamics*\*\RoleTailored Client]
2020-03-25T13:54:58.4805327Z *** Searching for Service-FinSql in Registry
2020-03-25T13:54:58.8029501Z *** Service-FinSql not found in registry: [HKLM:\SOFTWARE\Wow6432Node\Microsoft\Microsoft Dynamics*\*\Service]
2020-03-25T13:54:58.8576432Z *** Searching for RoleTailoredClient-FinSql in FileSystem
2020-03-25T13:54:58.8785051Z *** Found FinSql path: [C:\Program Files (x86)\Microsoft Dynamics NAV\140\RoleTailored Client\finsql.exe]
2020-03-25T13:54:58.9269198Z C:\Program Files (x86)\Microsoft Dynamics NAV\140\RoleTailored Client\finsql.exe
2020-03-25T13:55:05.9782847Z *** FOB File filter: C:\DevOps_Host\agentMeyerParkett\_work\1\s\objects.fob
2020-03-25T13:55:05.9930163Z *** Import Action: Overwrite
2020-03-25T13:55:05.9940978Z *** Start processing FOB: [objects.fob]
2020-03-25T13:55:06.0007381Z *** Transfer FOB to session: [objects.fob]
2020-03-25T13:56:41.3790158Z *** Import FOB: [objects.fob] in [NAV]
2020-03-25T13:56:41.3810823Z   * DatabaseServer   : localhost\SQLEXPRESS
2020-03-25T13:56:41.3819760Z   * Database Instance: SQLEXPRESS
2020-03-25T13:56:41.3833044Z   * Database Name    : Cronusat
2020-03-25T13:56:41.3836918Z   * Management Port  : 7045
2020-03-25T13:56:41.4772583Z *** Running command: & "C:\Program Files (x86)\Microsoft Dynamics NAV\140\RoleTailored Client\finsql.exe" --% Command=ImportObjects,ImportAction=Overwrite,SynchronizeSchemaChanges=Yes,File="C:\Run\objects.fob",LogFile="C:\Users\ContainerAdministrator\AppData\Local\Temp\NavIde\6412666d-b439-40ff-b1b2-e4697468b70f\objects.log",ServerName="localhost\SQLEXPRESS",generatesymbolreference=1",Database="Cronusat",NavServerName="localhost",NavServerInstance="NAV",NavServerManagementport=7045 | Out-Null
2020-03-25T13:56:41.6560083Z ##[error]!
2020-03-25T13:56:41.7107677Z ##[section]Finishing: ALOpsFobImport
```

this also happens when I import the Testtoolkit
```
2020-03-25T13:54:13.3238099Z ##[section]Starting: Import Testtoolkit
2020-03-25T13:54:13.3392785Z ==============================================================================
2020-03-25T13:54:13.3393394Z Task         : ALOps App Test
2020-03-25T13:54:13.3393629Z Description  : ALOps App Test
2020-03-25T13:54:13.3393834Z Version      : 1.428.1141
2020-03-25T13:54:13.3394089Z Author       : Hodor
2020-03-25T13:54:13.3394378Z Help         : Run Business Central Test-Suite and collect results.
2020-03-25T13:54:13.3394721Z ==============================================================================
2020-03-25T13:54:15.3514737Z *** Validate configuration
2020-03-25T13:54:15.8354740Z *** Importing required PS-Functions
2020-03-25T13:54:16.1844929Z *** Working folder: C:\DevOps_Host\agentMeyerParkett\_work\1\s
2020-03-25T13:54:16.1851922Z ***
2020-03-25T13:54:16.1860055Z *** Prepare Docker Session
2020-03-25T13:54:16.3527005Z *** Connect Docker Session
2020-03-25T13:54:16.3608804Z *** Initiate Docker Session
2020-03-25T13:54:16.8931883Z *** Copy Modules to Docker
2020-03-25T13:54:22.4252388Z *** Setup Docker Session
2020-03-25T13:54:22.4402921Z *** Import Modules in Docker Session
2020-03-25T13:54:22.4928345Z *** Import UI Client [C:\Run\DevOps\BCAppTest\bin\Microsoft.Dynamics.Framework.UI.Client.dll]
2020-03-25T13:54:22.4949757Z VERBOSE: Loading module from path 'C:\Run\DevOps\BCAppTest\bin\Microsoft.Dynamics.Framework.UI.Client.dll'.
2020-03-25T13:54:27.5678141Z *** Import NewtonSoft JSON [C:\Program Files\Microsoft Dynamics NAV\140\Service\Newtonsoft.Json.dll]
2020-03-25T13:54:27.5822711Z *** Import Modules in Docker Session: class_ClientContext
2020-03-25T13:54:29.9400874Z *** Setup BC Credentials in Docker
2020-03-25T13:54:29.9504788Z *** Prepare FinSql
2020-03-25T13:54:30.2171353Z *** Searching for Client-FinSql in Registry
2020-03-25T13:54:30.2176382Z *** Client-FinSql not found in registry: [HKLM:\SOFTWARE\Wow6432Node\Microsoft\Microsoft Dynamics*\*\RoleTailored Client]
2020-03-25T13:54:30.2181105Z *** Searching for Service-FinSql in Registry
2020-03-25T13:54:30.2185877Z *** Service-FinSql not found in registry: [HKLM:\SOFTWARE\Wow6432Node\Microsoft\Microsoft Dynamics*\*\Service]
2020-03-25T13:54:30.2190088Z *** Searching for RoleTailoredClient-FinSql in FileSystem
2020-03-25T13:54:30.2212352Z *** Found FinSql path: [C:\Program Files (x86)\Microsoft Dynamics NAV\140\RoleTailored Client\finsql.exe]
2020-03-25T13:54:30.2612068Z C:\Program Files (x86)\Microsoft Dynamics NAV\140\RoleTailored Client\finsql.exe
2020-03-25T13:54:37.0412269Z *** Import TestSuite Objects
2020-03-25T13:54:37.5649858Z *** Import TestSuite Page FOB: pstesttoolpage_auto.fob
2020-03-25T13:54:37.6259991Z   * DatabaseServer   : localhost\SQLEXPRESS
2020-03-25T13:54:37.6288243Z   * Database Instance: SQLEXPRESS
2020-03-25T13:54:37.6318926Z   * Database Name    : Cronusat
2020-03-25T13:54:37.6346919Z   * Management Port  : 7045
2020-03-25T13:54:37.6409694Z *** Running command: & "C:\Program Files (x86)\Microsoft Dynamics NAV\140\RoleTailored Client\finsql.exe" --% Command=ImportObjects,ImportAction=Overwrite,SynchronizeSchemaChanges=Force,File="C:\Run\DevOps\BCAppTest\CAL\pstesttoolpage_auto.fob",LogFile="C:\Users\ContainerAdministrator\AppData\Local\Temp\NavIde\97bb5ef6-6744-409c-acdd-5c27d8bf9212\pstesttoolpage_auto.log",ServerName="localhost\SQLEXPRESS",generatesymbolreference=0",Database="Cronusat",NavServerName="localhost",NavServerInstance="NAV",NavServerManagementport=7045 | Out-Null
2020-03-25T13:54:38.0256679Z *** Running command: & "C:\Program Files (x86)\Microsoft Dynamics NAV\140\RoleTailored Client\finsql.exe" --% Command=ImportObjects,ImportAction=Overwrite,SynchronizeSchemaChanges=Force,File="C:\TestToolKit\CALTestCodeunits.AT.fob",LogFile="C:\Users\ContainerAdministrator\AppData\Local\Temp\NavIde\124b520c-b31e-4757-85c2-4a2088df8f5c\CALTestCodeunits.AT.log",ServerName="localhost\SQLEXPRESS",generatesymbolreference=0",Database="Cronusat",NavServerName="localhost",NavServerInstance="NAV",NavServerManagementport=7045 | Out-Null
2020-03-25T13:54:38.3196346Z *** Import TestToolKit FOB: CALTestRunner.fob
2020-03-25T13:54:39.1701489Z *** Running command: & "C:\Program Files (x86)\Microsoft Dynamics NAV\140\RoleTailored Client\finsql.exe" --% Command=ImportObjects,ImportAction=Overwrite,SynchronizeSchemaChanges=Force,File="C:\TestToolKit\CALTestRunner.fob",LogFile="C:\Users\ContainerAdministrator\AppData\Local\Temp\NavIde\c369e45c-8fc7-4ba7-b336-60adcd8fee45\CALTestRunner.log",ServerName="localhost\SQLEXPRESS",generatesymbolreference=0",Database="Cronusat",NavServerName="localhost",NavServerInstance="NAV",NavServerManagementport=7045 | Out-Null
2020-03-25T13:54:40.0528445Z ##[error]!
2020-03-25T13:54:44.1458685Z ##[section]Finishing: Import Testtoolkit
```

**Comment by waldo1001** [comment]
I have to ask - did you try with hyperv isolation?  This is exactly the error that we got because of the February Update ...

**Comment by PatrickGrabensteiner** [comment]
The Februar update is not installed on the system, it has worked before the latest update of ALOps

**Comment by waldo1001** [comment]
I understand that - and I'd like to look into it, but the 20 agents we're using are just fine - this is the only report of a failure after the update.  So I'd like to investigate by letting you check a few things - including hyperv isolation? Could you test?  And send the entire pipeline as well, if that doesn't help?

**Comment by AdminHodor** [comment]
Dear @PatrickGrabensteiner ,

Could you please try again? We released a new update "1.428.1157" that addresses this bug. 
This bug gave the same results as with the February update, but the source of the issues was something different. 

Thanks again for the great feedback!

Kind regards,
